ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -2407,3 +2407,34 @@ def test_torch_sinc(
         on_device=on_device,
         input=x[0],
     )
+
+# sigmoid
+@handle_frontend_test(
+    fn_tree="torch.sigmoid",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+)
+def test_torch_sigmoid(
+    *,
+    dtype_and_x,
+    as_variable,
```

Comment:
We had a small refactor, you have to refactor the tests according to this.
https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py#L29
Thanks!

Reply:
Done !